pkg/dagger.io/dagger/core/fs.cue

```diff
@@ -92,6 +92,29 @@ import "dagger.io/dagger"
 	dest: string
 }
 
+// Remove file or directory from a filesystem tree
+#RmFile: {
+	$dagger: task: _name: "RmFile"
+
+	// Input filesystem tree
+	input: dagger.#FS
+
+	// Path to delete (handle wildcard)
+	// (e.g. /file.txt or /*.txt)
+	path: string
+
+	// Allow wildcard selection
+	// Default to: true
+	allowWildcard: *true | bool
+
+	// Do not report error when file do not exists
+	// Default to: true
+	allowNotFound: *true | bool
```

Comment:
I think can just skip support for this for now based on what you found here: https://github.com/dagger/dagger/issues/2408#issuecomment-1122381170

Reply:
I removed the option and commented the code in the handler in https://github.com/dagger/dagger/pull/2422/commits/e39926bccdb5ab6328c32cfc67f34574111c1d50